crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -87,15 +98,16 @@ pub async fn create(
     let mut items: serde_json::Value = serde_json::from_str(&contents)
         .map_err(|_| anyhow!("Failed to parse accounts file at {accounts_file_path}"))?;
 
-    let network_value = get_network(network)?.get_value();
+    let network_name = chain_id_to_network_name(chain_id);
+    println!("Detected chain_id: {chain_id:#x}");
```

Comment:
Perhaps this is something, that should be shown globally (and not only for account create subcommand)?

I know we talked about it, but right now I'm also wondering if we should include this info at all - maybe it should rather be available through https://github.com/foundry-rs/starknet-foundry/issues/278 only ? 

what do you think?

Reply:
So you think about putting some non-trivial stuff in there as well? for example `chain_id` that would be fetched from the url

Reply:
Yup, exactly. Basically _all the important_ things config-wise.